project/Dependencies.scala

```diff
@@ -12,7 +12,7 @@ object Dependencies {
   // https://github.com/sbt/sbt/issues/4531
   private val azureStorageBlobNioV = "12.0.0-beta.19"
   private val azureIdentitySdkV = "1.9.0-beta.2"
-  private val azureAppInsightsV = "3.4.12"
+  private val azureAppInsightsLogbackV = "2.6.4"
```

Comment:
Do we want to leave a comment here warning the future upgrade-hungry developer about consequences of bumping this version?

Reply:
Yeah, that's a good idea.